source/frontend/vite.config.ts

```diff
@@ -70,7 +70,7 @@ export default defineConfig({
         "base-uri 'self'; \
          default-src 'self'; \
          script-src 'self' blob: 'sha256-8ZgGo/nOlaDknQkDUYiedLuFRSGJwIz6LAzsOrNxhmU='; \
-         connect-src 'self' http://localhost:*/ https://collector-3cd915-dev.apps.silver.devops.gov.bc.ca/ https://www.arcgis.com/sharing/rest/ https://tiles.arcgis.com/ https://maps.gov.bc.ca/arcgis/rest/ https://server.arcgisonline.com/ArcGIS/rest/ https://dev.loginproxy.gov.bc.ca/ https://dev-pims.th.gov.bc.ca/api/ https://openmaps.gov.bc.ca/ https://delivery.apps.gov.bc.ca/ https://apps.gov.bc.ca/ https://maps.th.gov.bc.ca/ https://gov.bc.ca/ https://dev-maps.th.gov.bc.ca/; \
+         connect-src 'self' http://localhost:*/ https://collector-3cd915-dev.apps.silver.devops.gov.bc.ca/ https://www.arcgis.com/sharing/rest/ https://tiles.arcgis.com/ https://maps.gov.bc.ca/arcgis/rest/ https://server.arcgisonline.com/ArcGIS/rest/ https://dev.loginproxy.gov.bc.ca/ https://dev-pims.th.gov.bc.ca/api/ https://openmaps.gov.bc.ca/ https://delivery.apps.gov.bc.ca/ https://apps.gov.bc.ca/ https://maps.th.gov.bc.ca/ https://gov.bc.ca/ https://dev-maps.th.gov.bc.ca/ https://parcelmapbc.ltsa.ca/; \
```

Comment:
I don't this we need this CSP change anymore since we are no longer connecting to the LTSA arcgis service - right?